pkg/mysql/mysql.go

```diff
@@ -16,17 +16,19 @@ import (
 )
 
 const (
-	ComponentName    = "mysql"
-	DataVolumeName   = "datadir"
-	DataMountPath    = "/var/lib/mysql"
-	CustomConfigKey  = "my.cnf"
-	configVolumeName = "config"
-	configMountPath  = "/etc/mysql/config"
-	credsVolumeName  = "users"
-	CredsMountPath   = "/etc/mysql/mysql-users-secret"
-	tlsVolumeName    = "tls"
-	tlsMountPath     = "/etc/mysql/mysql-tls-secret"
-	BackupLogDir     = "/var/log/xtrabackup"
+	ComponentName     = "mysql"
+	DataVolumeName    = "datadir"
+	DataMountPath     = "/var/lib/mysql"
+	CustomConfigKey   = "my.cnf"
+	configVolumeName  = "config"
+	configMountPath   = "/etc/mysql/config"
+	credsVolumeName   = "users"
+	CredsMountPath    = "/etc/mysql/mysql-users-secret"
+	mysqlshVolumeName = "mysqlsh"
+	mysqlshMountPath  = "/.mysqlsh"
```

Comment:
i think this is okay but if we don't want to mount an empty dir for this, we can use `MYSQLSH_USER_CONFIG_HOME` to change this path